Plugin/Holacash/Model/Payment.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+namespace Signifyd\Connect\Plugin\Holacash\Model;
+
+use Signifyd\Connect\Helper\PurchaseHelper;
```

Comment:
@ebanolopes Hi Ebano! Can you help us with a review of this PR please? Thanks a lot :)